data/json/mutations/mutations.json

```diff
@@ -209,16 +209,6 @@
       { "part": "torso", "neutral": 10 }
     ]
   },
-  {
-    "type": "mutation",
-    "id": "PARKOUR",
```

Comment:
You need to migrate this trait to a proficiency.

Reply:
That would be prof_parkour in misc.josn

Reply:
But for existing players with this mutation, it will be an error on load and vanish. 
See src/savegame_json.cpp:680 for an existing example.

Reply:
Trait has been migrated to a proficiency by replacing PROF_HELI_PILOT in your example with PARKOUR. I figured this is okay since it was labelled for removal after 0.F. In this case, this little bit of code would be labelled for removal for 0.G.